### ISSUE TITLE
Unable to uninstall on unsupported CPU

### ISSUE THREAD

**Issue by Lord-Mustang** [intel/haxm#406]
**Describe the Bug**

Summary: 
It appears to be impossible to uninstall HAXM on an unsupported CPU.

**Host Environment**

* HAXM version: unknown

* Host OS version: Windows 10.0.19043

* Host OS architecture: x86\_64

* Host CPU model: AMD Ryzen 5 5600X

* Host RAM size: 32 GB

**To Reproduce**

1. Install Android Studio, Android AVD and Intel HAXM on a supported system
2. Switch the CPU to a different vendor that is not supported
3. Try to uninstall Intel HAXM through the control panel
4. Notice the inability to uninstall HAXM

**Expected Behavior**

Expectation: 
The ability to uninstall HAXM on unsupported CPUs

**Reproducibility**

100%

**Diagnostic Information**

Screenshots: 
![image](https://user-images.githubusercontent.com/33723142/132991857-3b731266-92fc-44bc-8a40-d19e547e5589.png)
![image](https://user-images.githubusercontent.com/33723142/132991864-f65c5a20-8bb1-4de1-8d95-9d93734030ea.png)
![image](https://user-images.githubusercontent.com/33723142/132991869-202d8453-3560-4d7a-afd6-2f6d831fd611.png)

**Comment by wcwang** [comment]
Thanks for your report. We will check this issue immediately.

**Comment by nevilad** [comment]
HAXM does not support AMD CPUs.

**Comment by wayne-ma** [comment]
Maybe you can switch to Intel CPU for uninstalling HAXM, then switch back to your AMD CPU.

**Comment by miklosakos** [comment]
> Maybe you can switch to Intel CPU for uninstalling HAXM, then switch back to your AMD CPU.
Yes, that's definitely a doable workaround. Let me buy an Intel PC just for that.
@wcwang has this issue been fixed yet?
